pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -8,11 +8,12 @@ function omf.repo.pull
   set -l repo_dir $argv[1]
 
   function omf.repo.git -V repo_dir
-    command git --git-dir "$repo_dir"/.git --work-tree "$repo_dir" $argv
+    command git -C "$repo_dir" $argv
```

Comment:
No problem, but let's -  big pretty please - decide what is the oldest `git` version OMF supports (sometime before OMFv1.0).
In #208 @bpinto said older OS X versions come with it's git 1.**8.7** and, you (Mr. @derekstavis :bowtie:) said that the installer uses shallow clones, and hence OMF requires even 1.**9**.
So let's get this confusion out of 

btw: I really hope, for the last bit of respect I have left for Apple, that upgrading Git to v2+ on those older OS X versions is a piece of cake and doesn't require Homebrew or anything like that.